app/controllers/ems_common.rb

```diff
@@ -656,6 +657,10 @@ def retrieve_openstack_security_protocols
     [[_('SSL without validation'), 'ssl'], [_('SSL'), 'ssl-with-validation'], [_('Non-SSL'), 'non-ssl']]
   end
 
+  def retrieve_openstack_amqp_security_protocols
+    [[_('Non-SSL'), 'non-ssl']]
```

Comment:
Might be nice to have a comment here saying that OSP 8 doesn't support SSL for AMQP ... won't hold up the merge on this though